src/components/Network.js

```diff
@@ -129,10 +145,12 @@ class Network extends React.Component {
             </div>
             <ul className="list-group list-group-flush">
               <li className="list-group-item">
-                <div>
-                  Synced timestamp: {dateFormatter.timestampToString(sync.synced_timestamp)}<br />
-                  Latest timestamp: {dateFormatter.timestampToString(sync.latest_timestamp)}
-                </div>
+                {sync_v1 ? (
```

Comment:
I thought about adding the `state` information here. Not sure if it's important but I guess it would be good just to know the status of the sync.

As described in the design: 
`state: can be unknown, behind, in-sync, ahead, to indicate where a peer state is in relation to us`

Reply:
I agree, what do you prefer?
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/5104565/112647127-f24d8480-8e26-11eb-9be8-6c52f960a963.png">

<img width="997" alt="image" src="https://user-images.githubusercontent.com/5104565/112648579-6dfc0100-8e28-11eb-8085-5637d645cf78.png">

Reply:
I think we shouldn't show 'in-sync'. We should have a pretty format like 'In sync'

Reply:
Done